README.md

```diff
@@ -11,6 +11,12 @@ allow the execution of classic Hello World program.
 | OP_ADD | sum two operands |
 | OP_SUB | subtracts two operands |
 | OP_PRINT | print |
+| OP_JLT | jump to specified address if operand 1 is less than 0 |
```

Comment:
Wanted to keep text similar to JMP.

> jump to specified address

I think it is better off to write as:

> jump to specified address in operand 2 if operand 1 is less than 0 

rather than

> jump to specified address if operand 1 is less than 0 

How do you think?